readme.md

```diff
@@ -330,12 +330,12 @@ We could create a routes module that defines our index route. Let's create a `co
 
 ```js
 module.exports = {
-  index: function( req, res ){
-    var numberOfBottles = parseInt(req.params.numberOfBottles) || 99;
-    var next = numberOfBottles - 1;
+  index( req, res ){
+    var bottles = parseInt(req.params.numberOfBottles) || 99;
+    var next = bottles - 1;
     res.render('index',{
-      bottles: numberOfBottles,
-      next: next
+      bottles,
+      next
     });
```

Comment:
maybe reduce to 1-liner since it isn't key-value pairs from *object literal property value shorthand*.(had to look up what that feature was)